src/todolist/interactive.js

```diff
@@ -0,0 +1,44 @@
+export default class Interactive {
+  constructor() {
+    this.toDoList = JSON.parse(localStorage.getItem('todo')) || [];
+  }
+
+  // update status
+  updateStatus(checkbox, index) {
+    checkbox.addEventListener('click', (event) => {
+      if (event.currentTarget.checked) {
+        checkbox.nextElementSibling.classList.add('text-decoration-line-through');
+        this.toDoList[index].completed = true;
+        localStorage.setItem('todo', JSON.stringify(this.toDoList));
+      } else {
+        if (checkbox.nextElementSibling.classList.contains('text-decoration-line-through')) {
+          checkbox.nextElementSibling.classList.remove('text-decoration-line-through');
+        }
+        this.toDoList[index].completed = false;
+        localStorage.setItem('todo', JSON.stringify(this.toDoList));
```

Comment:
Great job so far but consider the following changes :

- [ ] please consider using a function to implement `localStorage.setItem('todo', JSON.stringify(this.toDoList));` to avoid code repetition.